docs/pages/Nakama/client.mdx

```diff
@@ -8,7 +8,7 @@ While the Nakama console is helpful with testing, debugging, and managing your b
 
 ## Simple JS Client
 
-The [starter-game-template has a simple node JS client](https://github.com/Argus-Labs/starter-game-template/tree/main/.jsclient) that simply joins the global receipt-broadcasting match and prints those receipts directly to the standard output.
+The [starter-game-template has a simple node JS client](https://github.com/Argus-Labs/starter-game-template/tree/main/.jsclient) that simply echos transaction receipts directly to the standard output.
```

Comment:
This link no longer works because this file was deleted... :/ 

@technicallyty can you chime in on what we wanna do about this? We can remove this Simple JS Client from the docs for now but I think we should make a *serious* effort to have a gold standard Nakama client that demonstrates usage of every feature we have in JS and perhaps in another lang too.

Reply:
based on what we discussed in meeting, looks like we can just remove this for now and add it back in when we/I write a JS client

Reply:
I removed the JS section